c2rust-analyze/tests/filecheck.rs

```diff
@@ -60,7 +60,13 @@ fn filecheck() {
     let filecheck_bin = env::var_os("FILECHECK")
         .map(PathBuf::from)
         .or_else(|| detect_filecheck().map(|it| it.to_owned()))
-        .unwrap_or_else(|| panic!("FileCheck not found - set FILECHECK=/path/to/FileCheck"));
+        .unwrap_or_else(|| {
+            PathBuf::from(
+                invoke_command(find_llvm_config().as_deref(), &["--bindir"])
+                    .expect("llvm-config not found"),
+            )
```

Comment:
Need to append `/FileCheck`.

Reply:
fixed in [7706f76](https://github.com/immunant/c2rust/pull/767/commits/7706f76352351523abae3a26f9f58859e882cd6f)